test/test_data/output_formatter.medium.fortio.gold

```diff
@@ -476,6 +476,8 @@
    }
   ]
  },
+ "BytesSent": "3528",
```

Comment:
Is there a test case that covers this change?

Reply:
Yes, the integration tests check rx/tx byte counts e.g: https://github.com/envoyproxy/nighthawk/blob/1ef11bb53556c856e043ce64a1d1108635a279ad/test/integration/test_integration_basics.py#L32